std/file.d

```diff
@@ -862,7 +862,7 @@ unittest
 version(StdDdoc) void getTimesWin(R)(R name,
                                   out SysTime fileCreationTime,
                                   out SysTime fileAccessTime,
-                                  out SysTime fileModificationTime)
+                                  out SysTime fileModificationTime) @safe
```

Comment:
I've never seen a case where one creates a template function without a body for ddoc. Generally, they are implemented with either a dummy return or an empty body.

Technically, the inference depends on calls to Windows functions, and however we mark those is what should define the attributes. I don't know what ddoc does for inferred attributes, so I'm not sure if it really matters whether you add the attribute directly or infer it. However, it *should* be the same for both ddoc and real version.